package.json

```
@@ -1,68 +1,62 @@
 {
-  "name": "ovh-angular-chatbot",
-  "version": "1.0.4",
-  "description": "chatbot component for the ovh chatbot",
+  "name": "ng-ovh-chatbot",
+  "version": "2.0.0",
```

`yarn version` will handle the bump.

```suggestion
  "version": "1.0.4",
```